search-sentiments-map/public/scripts/search-topic-script.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the 'License'); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+/**
+ * Updates the current search topic for the given country and search topic.
+ */
+function searchTopic() {
+  let topic = document.getElementById("search-topic").value;
+  let topTrends = getTopTrends();
+  // Maximum of 3 countries. If given more, only take first 3.
+  let countries = $('#country-select').val().slice(0, 3);
+
+  // Get new search results when given a topic and at least 1 country.
+  if (topic.length !== 0 && countries.length !== 0) {
+    setUserSearchTopic(topic, countries);
+  }
```

Comment:
General rule for things like this is to do an early return.

So you check if topic is empty, or theres no countries, and then return.

After that, in the normal code flow you would call `setUserSearchTopic`.

It wont make much of a difference in this specific case since the function is trivial, but its better practice.